### PR TITLE
Institution-api Kubernetes config

### LINKED ISSUE #1719: Create deployment artifacts for institutions API microservice
Create the following in the `institutions-api` project: 
* `Helm` Chart to deploy the microservice to `Kubernetes` cluster
* `Jenkinsfile` to build and deploy current branch to `Kubernetes` cluster

For reference, the `check-digit` and `hmda-platform` projects have these files already implemented (see project's root folder as well as _kubernetes_ directory)

### PR DESCRIPTION
**Pull request by nickgrippin**
Closes #1719

### CODE REVIEW

**Comment by jmarin** [comment]
I don't see any configuration for `PostgreSQL` server connection

File: kubernetes/institutions-api/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      labels:
+        app: {{ template "institutions-api.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: POSTGRES_HOST
```
Comment (jmarin) [inline kubernetes/institutions-api/templates/deployment.yaml]: Have you tried running this? The `PostgreSQL` environment variables don't seem to match the ones used in this microservice (`PG_HOST`, `PG_PORT`, `PG_USER`, `PG_PASSWORD`, `PG_DATABASE`)

**Comment by jmarin** [comment]
@nickgrippin Can ou add `Cassandra` configuration for this microservice as well? I'm getting tied up on other aspects of `k8s` on my branch and I don't want to hold up this PR because of that. Thanks

**Comment by jmarin** [comment]
@nickgrippin There are some merge conflicts with this one
Also, can you add the `Cassandra` configuration so that this pulls `Cassandra` username / password from secrets?. See configuration [here](https://github.com/cfpb/hmda-platform/pull/1923/files#diff-13a1c21b0ac339e77bf1321e48691c2aR8) and [here](https://github.com/cfpb/hmda-platform/pull/1923/files#diff-2dcd9c988b36056d7f9369655c7e1235R33)
Let's not worry about pulling in a `Cassandra` subchart for local development for now so that we can get this merged.

**Comment by nickgrippin** [comment]
Sounds good, I'll have an update pushed shortly

**Comment by jmarin** [comment]
Also, one more thing about this one. The `PostgreSQL` credentials should be made as part of a `Kubernetes` secret (see the `Cassandra` configuration in my PR mentioned above). Config maps are fine for the host name and other configuration which is fine to store un-encrypted

**Comment by jmarin** [comment]
Don't have time to test it right now, but LGTM

**Pull request merged**